### ISSUE TITLE
Allow for serial accepting of tasks in process list

### ISSUE THREAD

**Issue by aetherfaerber** [kitodo/kitodo-production#6327]
**Is your feature request related to a problem? Please describe.**
It is possible to accept a task through the process list. (Show details, press bell-button.) Once you accept the task, Kitodo opens the task, i.e. moves to the task area with the task opened. If you now mark the task as done you get forwarded to the task list (Aufgaben). If you want to accept another task from the process list, you will first have to move back to the process list. But oh no, you get a fresh process list and all your filters have been removed and you have to re-enter them one by one.

**Describe the solution you'd like**
When I open a task from the process list I should get back to the process list after closing the task. Filters on the process list should be preserved as they are when opening and closing a task from the task list.

**Describe alternatives you've considered**
Using the task list seems like a workable alternative but doing it through the process list should be possible as well because some users just prefer the process list (also filtering is more effective here).

**Additional context**
Could also be split into a bug (filter gets lost) and a feature request (forward back to process list).

**Comment by BartChris** [comment]
What do you think about the option of tracking the "source view" (the page the user originally came from) and redirecting back to it (so either process or task list), if technically feasible?

**Comment by aetherfaerber** [comment]
> What do you think about the option of tracking the "source view" (the page the user originally came from) and redirecting back to it (so either process or task list), if technically feasible?

Yes, this sounds the way users would expect things to work. I don't think you should always be redirected to the process list but to the list you came from when you accepted the task.
